userstatus/userstatuswwu/classes/userstatuswwu.php

```diff
@@ -292,14 +292,9 @@ private function order_delete() {
                 // In case the user is not in the zivmemberlist and was suspended for longer than one year he/she ...
                 // ... is supposed to be deleted.
                 if ($moodleuser->timestamp < $timestamp - 31622400) {
-                    // Check whether the user is again listed.
-                    if (!array_key_exists($moodleuser->username, $this->zivmemberlist)) {
```

Comment:
Ich versuche das schnell zu reviewn, deswegen übersehe ich vielleicht etwas, aber brauchen wir den check nicht für den Fall, dass ein Nutzer suspended war (für 1-2 Monate nicht in der groups.txt war) Und nun wieder drin ist?

Reply:
Hey, das wird schon in Zeile 291 überprüft :)

Reply:
🤦